arangod/Metrics/Counter.h

```diff
@@ -40,8 +40,8 @@ class Counter final : public Metric {
   ~Counter() final;
 
   [[nodiscard]] std::string_view type() const noexcept final;
-  void toPrometheus(std::string& result, std::string_view globals,
-                    std::string_view alternativeName) const final;
+  void toPrometheus(std::string& r, bool first,
```

Comment:
Because its clearly that is "result", also we have non convenient clang-format with restriction 80 symbols line width.

Maybe better add replace std::string to using Result = std::string?